contracts/governance/StakingRewards/StakingRewards.sol

```diff
@@ -149,4 +151,22 @@ contract StakingRewards is StakingRewardsStorage, Initializable {
 		require(SOV.transfer(_receiver, _amount), "transfer failed");
 		emit RewardWithdrawn(_receiver, _amount);
 	}
+
+	/**
+	 * @notice Get staker's current accumulated reward
+	 * @dev Collecting rewards can yeild slightly different values as there are mandatory checks
+	 * while withdrawing:
+	 * 1. User cannot withdraw within 14 days from the start/last withdrawn date
+	 * 2. The rewards can only be processed for a max duration at a time
+	 * @return The accumulated reward
+	 */
+	function getStakerCurrentReward() external view returns (uint256 amount) {
```

Comment:
pls use this function in collectRewards() else it is code duplication

Reply:
Done